paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PaymentSheetScreen.kt

```diff
@@ -274,7 +274,7 @@ private fun PaymentSheetContent(
         if (mandateText?.showAbovePrimaryButton == true) {
             Mandate(
                 mandateText = mandateText.text?.resolve(),
-                modifier = Modifier.padding(horizontal = horizontalPadding),
+                modifier = Modifier.padding(horizontal = horizontalPadding).padding(bottom = 8.dp),
```

Comment:
this padding is also set for the main screen content. We need to include it here so that there is consistent padding between either the content or the mandate and the error message (which is directly below the mandate when shown)